src/index.js

```diff
@@ -916,24 +916,34 @@ const announcement = new Announcement();
             // No previous scrape results yet, do not proceed
             if (typeof election.prev === 'undefined') return;
 
-            // Previously had no primary, but after re-scraping there is one
+            // Previously had no primary, but after rescraping there is one
             if (!announcement.hasPrimary && election.datePrimary != null) {
                 announcement.initPrimary(election.datePrimary);
                 await room.sendMessage(`There will be a primary phase before the election now, as there are more than ten candidates.`);
             }
 
-            // After re-scraping the election was cancelled
+            // After rescraping the election was cancelled
             if (election.phase === 'cancelled' && election.prev.phase !== election.phase) {
                 await announceCancelled(election);
-                return;
             }
 
-            // After re-scraping we have winners
+            // After rescraping we have winners
             else if (election.phase === 'ended' && election.prev.arrWinners.length != election.arrWinners.length && election.arrWinners.length > 0) {
                 await announceWinners(election);
+
+                // No more need to scrape the election or greet the room
+                BotConfig.scrapeIntervalMins = 999999;
```

Comment:
Yeah, sorry, I was unclear about what I mean - I didn't want to make it `-1` and then pass that unconditionally to `setTimeout` but rather make it a special value that is then guarded against before setting the timeout. It would also be idiomatic since `lastMessageTime` also uses `-1` as a special value. In any case, this is just a note that we could maybe think about after more pressing matters